pkgs/tools/audio/mpdris2/default.nix

```diff
@@ -17,8 +20,7 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ autoreconfHook ];
   buildInputs = [ intltool pythonPackages.wrapPython ];
-  propagatedBuildInputs = with pythonPackages; [ python pygtk dbus-python  ];
-  pythonPath = with pythonPackages; [ mpd pygtk dbus-python notify mutagen ];
+  pythonPath = with pythonPackages; [ pygobject3 mpd2 dbus-python mutagen ];
   postInstall = "wrapPythonPrograms";
```

Comment:
If you replace `stdenv.mkDerivation` with `python3.pkgs.buildPythonApplication`, you can drop `wrapPython`.

Will require using `propagatedBuildInputs` instead of `pythonPath` and adding `format = "other";` since Autotools are used.

Reply:
Switched to buildPythonApplication.